.github/workflows/preview.yml

```diff
@@ -40,6 +40,7 @@ jobs:
             --env XATA_PREVIEW=none \
             --build-env XATA_BRANCH=main \
             --env XATA_BRANCH=main \
+            --no-wait \
```

Comment:
My original action had the `--no-wait` but @snide wanted it to wait

Reply:
Great minds think alike!

I've added an extra step to wait for the vercel deploy. Hopefully it's more robust this way....

Reply:
Started a thread about the root cause here. 

https://xata-hq.slack.com/archives/C032EKCBUGM/p1692784605431259

Reply:
Reason not to have the no-wait was so that it didn't generate the comment before the preview completed (in the case where it errors)